samples/ABNF/tranquil.abnf

```diff
@@ -0,0 +1,61 @@
+; Source:  https://github.com/fjolnir/Tranquil
+; License: BSD 3 Clause License
+; Modified for standard compatibility by René Schwaiger
```

Comment:
Recall that these files are for the classifier: 99% of the time, they remain unread by human beings.